de.klayout.KLayout.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: de.klayout.KLayout
+runtime: org.kde.Platform
+runtime-version: "5.14"
+sdk: org.kde.Sdk
+command: klayout
+finish-args:
+- --share=ipc
+- --socket=x11
+- --socket=wayland
+- --filesystem=host
+- --device=dri
+modules:
+- name: gdbm
+  config-opts:
+  - --enable-libgdbm-compat
+  sources:
+  - type: archive
+    url: ftp://ftp.gnu.org/gnu/gdbm/gdbm-1.18.1.tar.gz
```

Comment:
Can you switch it to https?

Reply:
I switched it. The gdbm site doesn't have an https link, but switching to https worked.